### PR TITLE
Postgres Setup Improvements

### PR DESCRIPTION
**Pull request by murjax**
Postgres + Knex is complicated. Simplifying some processes so it's easier to setup.

### CODE REVIEW

File: server/package.json [hunk 1]
```diff
@@ -4,6 +4,14 @@
...
   "main": "index.js",
   "license": "MIT",
+  "scripts": {
+    "db:create": "node knex_create.js",
+    "db:drop": "node knex_drop.js",
+    "db:migrate": "knex migrate:latest",
+    "db:seed": "knex seed:run",
+    "db:reset": "node knex_drop.js && node knex_create.js && knex migrate:latest",
+    "db:setup": "node knex_drop.js && node knex_create.js && knex migrate:latest && knex seed:run"
+  },
```
Comment (murjax) [inline server/package.json:14]: I'm mimicking [Rails](https://guides.rubyonrails.org/active_record_migrations.html) commands for database management here. Open to ideas if there's clearer commands. One or two of these scripts will probably be mapped up to the root level package.json as build scripts for production.

File: server/package.json [hunk 2]
```diff
@@ -4,6 +4,14 @@
   "description": "Jax Pet Names API",
   "main": "index.js",
   "license": "MIT",
+  "scripts": {
+    "db:create": "node knex_create.js",
+    "db:drop": "node knex_drop.js",
+    "db:migrate": "knex migrate:latest",
```
Comment (murjax) [inline server/package.json:10]: I finally found the knex command for running _all_ migrations!

File: server/package.json (hunk 1 above)
Comment (JaxTurboNerd) [reply server/package.json:14]: Excellent.  I'll take a look and try to get Docker set up and give it a try.

File: server/package.json (hunk 2 above)
Comment (JaxTurboNerd) [reply server/package.json:10]: excellent!

**Review by JaxTurboNerd — APPROVED** [review]
Approved.  I tested the code in my local branch and everything worked perfectly.  README.md instructions were easy to follow to set up Docker.

**Pull request merged**